go/test/endtoend/vreplication/fk_test.go

```diff
@@ -88,7 +88,7 @@ func TestFKWorkflow(t *testing.T) {
 	}
 	targetKeyspace := "fktarget"
 	targetTabletId := 200
-	vc.AddKeyspace(t, []*Cell{cell}, targetKeyspace, shardName, initialFKTargetVSchema, initialFKSchema, 0, 0, targetTabletId, sourceKsOpts)
+	vc.AddKeyspace(t, []*Cell{cell}, targetKeyspace, shardName, initialFKTargetVSchema, "", 0, 0, targetTabletId, sourceKsOpts)
```

Comment:
This change to the test ensures the new behavior is applied. The schema in the test is composed of `parent` and `child` tables. Since `child < parent` lexicographically, the test fails without the new normalization.